src/hardware/motor.cpp

```diff
@@ -482,8 +501,8 @@ void StepperMotor::setState(MOTOR_STATE newState, bool clearErrors) {
                 case ENABLED:
 
                     // Drive the coils the current angle of the shaft (just locks the output in place)
-                    driveCoils(getAngle() - startupAngleOffset, COUNTER_CLOCKWISE);
-                    
+                    driveCoils(getAngle() - startupAngleOffset);
```

Comment:
A driveCoils call can't use angles, they have to use steps. You need a driveCoilsAngle

Reply:
Fixed in
https://github.com/CAP1Sup/Intellistep/pull/43/commits/9223bbed425705993372218e83d6d65d57c33c83